_layouts/landing_page.html

```diff
@@ -16,7 +16,11 @@
 
 	{% include footer.html %}
 
-	<script src="{{ "js/github-queries.js" }}"></script>
+	<!-- Existing script -->
+	<script src="{{ '/js/github-queries.js' | relative_url }}"></script>
+
+	<!-- for dynamic Latest News -->
+	<script src="{{ '/js/news-collect.js' | relative_url }}"></script>
```

Comment:
These comments are not relevant/helpful:


```suggestion
	<script src="{{ '/js/github-queries.js' | relative_url }}"></script>

	<script src="{{ '/js/news-collect.js' | relative_url }}"></script>
```